ui/app/components/secret-edit.js

```diff
@@ -1,27 +1,27 @@
-import Ember from 'ember';
```

Comment:
this has a lot removed because I moved the relevant functions to new components. I also added documentation, but did not make this a glimmer component because I ran into some issues.